tracer/src/WindowsInstaller/WindowsInstaller.wixproj

```diff
@@ -20,7 +20,7 @@
     <OutputName>datadog-dotnet-apm-2.2.0-$(Platform)</OutputName>
     <TracerHomeDirectory Condition="'$(TracerHomeDirectory)' == ''">$(MSBuildThisFileDirectory)..\bin\windows-tracer-home</TracerHomeDirectory>
     <DefineConstants>InstallerVersion=2.2.0;TracerHomeDirectory=$(TracerHomeDirectory);LibDdwafDirectory=$(LibDdwafDirectory)</DefineConstants>
-    <LibDdwafDirectory Condition="'$(LibDdwafDirectory)' == ''">$(LibDdwafDirectory)..\..\packages\libddwaf.1.0.7</LibDdwafDirectory>
+    <LibDdwafDirectory Condition="'$(LibDdwafDirectory)' == ''">$(LibDdwafDirectory)..\..\packages\libddwaf.1.0.17</LibDdwafDirectory>
```

Comment:
I think this is the third time this is referenced - should we move this to being an MSBuild variable in the top-level Directory.Build.props so we don't miss any in the future?

Reply:
They're already set in the Nuke build scripts, so this isn't really needed, it was just belt and braces, which is actually counter productive in this case. Fixed in 7fadb646d